src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/IntellisenseQuickInfoBuilder.cs

```diff
@@ -41,21 +42,48 @@ internal static class IntellisenseQuickInfoBuilder
                 firstLineElements.Add(new ImageElement(warningGlyph.GetImageId()));
             }
 
+            var elements = new List<object>();
             var descSection = quickInfoItem.Sections.FirstOrDefault(s => s.Kind == QuickInfoSectionKinds.Description);
             if (descSection != null)
             {
-                firstLineElements.Add(BuildClassifiedTextElement(descSection));
+                var isFirstElement = true;
+                foreach (var element in BuildClassifiedTextElements(descSection))
+                {
+                    if (isFirstElement)
+                    {
+                        isFirstElement = false;
+                        firstLineElements.Add(element);
+                    }
+                    else
+                    {
+                        elements.Add(element);
+                    }
```

Comment:
it's a draft, so feel free to ask me to wait.  However, i was surprised to see cahnges like this here...  are they really necessary if we're just classifying paramrefs?

Reply:
Only the last commit is specific to this pull request. The first comment links to the main PR.